sdk/storage/azure-storage-blob/azure/storage/blob/_shared_access_signature.py

```diff
@@ -340,7 +339,7 @@ def generate_account_sas(
             :dedent: 8
             :caption: Generating a shared access signature.
     """
-    sas = SharedAccessSignature(account_name, account_key)
+    sas = SharedAccessSignature(account_name, account_key, x_ms_version=kwargs.pop('api_version', None))
```

Comment:
Here, if you didn't set api version, then the api_version will be None

Reply:
Great catch - thanks!